sql-mutations/diplomat.sql

```diff
@@ -0,0 +1,3 @@
+delete
+  from "addresses"
+  where "addressId" = 161
```

Comment:
You'll want `returning *` here, and in all your mutations. That's not critical now, but it will be when we do full-stack development.